pages/searching/configuration.rst

```diff
@@ -0,0 +1,56 @@
+Search configuration
+--------------------
+
+Graylog allows customizing the options allowed to search queries, like limiting the time range users can select or configuring the list of displayed relative time ranges.
+
+.. image:: /images/searching/queries_search_configuration.png
+
+All search configuration settings can be customized using the web interface on the *System* -> *Configurations* page in the *Search configuration* section.
+
+
+Query time range limit
+^^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes the amount of data stored in Graylog is quite big and spans a wide time range (e. g. multiple years). In order to prevent normal users from accidentally running search queries which could use up lots of resources, it is possible to limit the time range that users are allowed to search in.
```

Comment:
suggestion to swap `normal user` to `daily user` or `average user`

Reply:
I've updated the word to `daily user` here: https://github.com/Graylog2/documentation/pull/800